### PR TITLE
Some_blog: controllers specs

### PR DESCRIPTION
**Pull request by ambrose-kibet**
## In this milestone we:
- [ ] Add tests for post conrooller
- [ ] Add test for posts conrolller

### CODE REVIEW

File: spec/requests/posts_request_spec.rb [hunk 1]
```diff
@@ -1,7 +1,29 @@
...
+  it 'should succesfully request for posts ' do
+    get '/users/13/posts/1'
+    expect(response.status).to eq(200)
+  end
+
+  it 'check the response body includes correct placeholder text' do
+    get '/users/13/posts/1'
+    expect(response.body).to include('<h1>placeholder for show template</h1>')
   end
 end
```
Comment (julie-ify) [inline spec/requests/posts_request_spec.rb]: - [ ] Kindly separate the index and show tests into different describe blocks for better code organization.

  ```
  describe 'GET /index' do
    ...
  end

  describe 'GET /show' do
    ...
  end
  ```

File: spec/requests/users_request_spec.rb [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+    expect(response.status).to render_template(:show)
+  end
+  it 'should succesfully request for users ' do
+    get '/users/1'
+    expect(response.status).to eq(200)
+  end
+  it 'check the response body includes correct placeholder text' do
+    get '/users/1'
+    expect(response.body).to include('<h1>user show placeholder</h1>')
   end
```
Comment (julie-ify) [inline spec/requests/users_request_spec.rb]: - [ ] Kindly separate the index and show tests into different describe blocks for better code organization.

  ```
  describe 'GET /index' do
    ...
  end

  describe 'GET /show' do
    ...
  end
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @ambrose-kibet,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
- [ ] Most of your tests are failing. You need to fix them. If you look at the second image you would see that your tests are mostly failing because the `id`s you are trying to refer to do not exist. You need to create users and posts within your test files and refer to their id: `user.id`/`post.id`.

  jfifp

<hr/>

  fopr


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_request_spec.rb (hunk 2 above)
Comment (ambrose-kibet) [reply spec/requests/users_request_spec.rb]: testrev

**Review by prolajumokeoni — APPROVED** [review]
Hi @ambrose-kibet 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- All changes requested have been implemented 🥇  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**